app/models/card.rb

```diff
@@ -1,5 +1,8 @@
 class Card < ActiveRecord::Base
-  before_validation :add_review_time
+  scope :rand_word, -> { order("RANDOM()").limit(1) }
+  scope :need_check, lambda { where('review_date <= ?', Time.now) }
```

Comment:
Use the new lambda literal syntax `-> {...}`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.